Composer/packages/server/src/controllers/auth.ts

```diff
@@ -42,13 +45,15 @@ async function getAccessToken(req: GetAccessTokenRequest, res: Response) {
   });
 }
 
-async function logOut(req, res) {
+function logOut(req, res) {
   try {
     authService.logOut();
-  } catch {
-    res.status(500);
+    res.status(200).end();
+  } catch (err) {
+    log('Error logging out:');
+    log(err);
+    res.status(500).end();
```

Comment:
nit: it might be useful to send the error object to the client `res.status(500).json(err).end()`

Reply:
Will do.